homework.js

```diff
@@ -0,0 +1,113 @@
+// 1.Create a function that will get the sum of the numbers between 1 and n and return the answer
+
+function summation(n)
+ {
+  let sum = 0;
+  for (let i = 1; i <= n; i++) 
+      {
+    sum += i;
+      }
+  return sum;
+}
+
+console.log( summation(5) );
+
+
+// 2. Create a program to get the sum of all the even numbers in a group 
+
+function summationEven(n)
+ {
+  let sum = 0;
+  for (let i = 1; i <= n; i++) 
+      {
+if(i % 2 == 0)
+    sum += i;
+      }
+  return sum;
+}
+
+console.log( summationEven(5) );
+
+// 3.Create a function to get the average of a group of numbers 
+
+avrg = [2, 2, 8, 4, ];
+
+for ( var i = 0; i < avrg.length; i ++){
+    var avg = (avrg[i] / avrg.length) * avrg.length
+}
+console.log(avg)
+
+// 4. Create a function to reverse the letters in a word 
+
+function reverse (n) {
+    ​
+    n.sort((a, b) => b - a);
```

Comment:
The `.sort()` method only works with arrays and doesn't work with strings.
You can, however, use a string method to turn the string into an array which is `.split("")`

But even if you did that and tried to apply `.sort()` it wouldn't reverse the word it would simply arrange the letters from `a` to `z`.

If you wanted to reverse the letters you can use the array method `.reverse()` which would reverse the elements of the array, then use `.join()` which will turn the array back into a string.